File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/01.First-Steps-In-Coding/02-exercise/09.fishTank.js
function fishTank(input) {

    const length = Number(input[0]);
    const width = Number(input[1]);
    const height = Number(input[2]);
    const percent = Number(input[3]);

    const capacityInLiters = length * width * height / 1000;
    const finalCapacity = capacityInLiters - capacityInLiters * (percent / 100);

    console.log(finalCapacity);

}
fishTank(["105", "77", "89", "18.5"]);

File: Python_Basics/Homeworks-And-Labs/01.first_steps_in_coding/02.exercise/09.fish_tank.py
length = int(input())
width = int(input())
height = int(input())
percent = float(input())

capacity = length * width * height
# Един литър вода се равнява на един кубичен дециметър/ 1л=1 дм3/. 
capacity_in_litters = capacity / 1000
capacity_in_litters -= capacity_in_litters * (percent / 100)

print(capacity_in_litters)
